.github/workflows/release-apps.yml

```diff
@@ -0,0 +1,164 @@
+name: "release-apps"
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-[a-z]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-[a-z]+.[0-9]+'
+
+jobs:
+  create-release:
+    permissions: write-all
+    environment: Android Service Runtime Release
+    runs-on: ubuntu-latest
+    outputs:
+      releaseId: ${{ steps.step1.outputs.id }}
+    steps:
+      - id: step1
+        uses: ncipollo/release-action@v1
+        with:
+          name: "Android Service Runtime ${{ github.ref_name }}"
+          body: |
+            See the assets to download this version and install.
+          prerelease: true
+          draft: true
+
+  release-manager-app:
+    permissions: write-all
+    environment: Android Service Runtime Release
+    needs: 
+      - create-release
+    runs-on: 'ubuntu-22.04' 
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - feature: ''
+            label: user
+          - feature: 'system_settings'
+            label: system
```

Comment:
Build manager app with default feature (for use as a user app, accessible via the app grid), or with `system_settings` (for use as a system app, accessible only via the system settings)